keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -127,7 +130,22 @@ class KeeperPlugin : Plugin<Project> {
         val androidJarRegularFileProvider = project.layout.file(androidJarFileProvider)
 
         appExtension.testVariants.configureEach {
-          val appVariant = testedVariant
+          val appVariant = unwrapTestedVariant(this)
```

Comment:
I think you can go back to accessing `testedVariant` directly here. `UnitTestVariant` isn't a subclass/interface of `TestVariant`.

Reply:
3d61945